### PR TITLE
Explore Page: Show logos in cards

### PR DESCRIPTION
**Pull request by arsenijesavic**
closes https://github.com/hypha-dao/dho-web-client/issues/1028

### CODE REVIEW

File: src/components/ipfs/ipfs-image-viewer.vue [hunk 1]
```diff
@@ -47,8 +48,13 @@ export default {
     }
   },
 
+  updated () {
+    console.log('HERE')
```
Comment (pugakn) [inline src/components/ipfs/ipfs-image-viewer.vue:52]: Please remove console logs @arsenijesavic

Comment (arsenijesavic) [reply src/components/ipfs/ipfs-image-viewer.vue]: 😬 removed

**Review by pugakn — COMMENTED** [review]
Using the filter on the explorer page causes logo images to break, can you check it before the merge? @arsenijesavic

**Comment by arsenijesavic** [comment]
@pugakn fixed!

**Review by pugakn — APPROVED** [review]

**Pull request merged**